docs/faq/can-i-host-webiny-on-my-laptop.md

```diff
@@ -0,0 +1,13 @@
+---
+id: can-i-host-webiny-on-my-laptop
+title: Can I host Webiny on my laptop?
+sidebar_label: Can I host Webiny on my laptop?
+---
+
+No, you can't.
```

Comment:
No, you can't.
->
The short answer is that you are unable to do so.

Reply:
fixed